airgun/views/host.py

```diff
@@ -199,6 +199,7 @@ class HostsView(BaseLoggedInView, SearchableViewMixin):
     export = Text(
         ".//a[contains(@class, 'btn')][contains(@href, 'hosts.csv')]")
     new = Text("//a[contains(@href, '/hosts/new')]")
+    select_all = Checkbox(locator="//input[@id='check_all']")
```

Comment:
I thought we should have a function that works on the table to select all option or select_all. I don't see that option in widgetastic :(